### PR TITLE
Route highlight made simple - UI-2452 , UI-2453 and UI-2457 

### PR DESCRIPTION
**Pull request by sheela-svmx**
Now route hightlight will just add a black layer on top of the existing route color with a 50% opacity so irrespective of the route color the high light would be a bit darker of that shade

screen shot 2018-09-06 at 3 45 16 pm
screen shot 2018-09-06 at 3 48 44 pm
screen shot 2018-09-06 at 3 51 50 pm

### CODE REVIEW

File: px-map-behavior-layer-geojson.es6.js [hunk 1]
```diff
@@ -335,6 +342,7 @@
      */
 
     _handleFeatureTapped(evt) {
+      let currentRouteColor = evt.target.options.color;
```
Comment (mallik-svmx) [inline px-map-behavior-layer-geojson.es6.js]: I don't quite get this, which part of the geoJson data its coming from? (I see the attached screenshots with green, purple etc., route segments) Again the above weight property for both objectToAppendWeight and objectToAppendHighlight is 5, but the screenshot shows white filled/pipe line.

Comment (joshsylvester) [reply px-map-behavior-layer-geojson.es6.js]: the white willed / pipe line is created with three layers - a layer that has a width of 5, with the original color that was passed in as property of the geoJson, another identical size layer (width: 5) with a 50% black layer (thereby darkening the line that is being highlighted) and a third layer in white, the width of the original line. As long as the GeoJson has the correct color for each segment, the highlight will base it's color on the line passed in with the GeoJson.

Comment (mallik-svmx) [reply px-map-behavior-layer-geojson.es6.js]: Thanks @joshsylvester. I got the layer 1 and 3.
Like in my comment, the geoJson part of option color is never in the json, I guess may be its default color/option?
                {
                  "type": "Feature",
                  "id": "cdf6ba8e-5b16-470b-a423-30a5930e4129",
                  "properties": {
                    "Drive Time": "42 minutes",
                    "Mileage": "25.05 miles"
                  },
                  "geometry": {
                    "type": "LineString",
                    "coordinates": [
                      [
                        -122.385787,
                        37.617906
                      ],
....
                    ]
                  }
                },

**Comment by sheela-svmx** [comment]
2 approvals merging

**Pull request merged**